mqtt/mqtt-broker/src/auth/authorization.rs

```diff
@@ -72,6 +72,7 @@ impl Authorizer for AllowAll {
 }
 
 /// Describes a client activity to authorized.
+#[derive(Clone, Debug)]
```

Comment:
Do you use these clones anywhere?

Reply:
you're right, probably I needed them at some point, but now I don't use them.